File: Day2.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
double m;
int t;
int x;
int fprice;       
cin>>m;       
cin>>t;
cin>>x;
double tax=m*t*0.01;
double tip=x*m*0.01;
double f=tax+tip+m;
fprice =round(f);
cout<<"The final price of the meal is $"<<fprice<<".";
return 0;
}

File: Day2.py
import math
m=float(input())
t=int(input())
x=int(input())
tip=m*t/100
tax=x*m/100
finalprice=tip+tax+m
a=round(finalprice,0)
ans=int(a)
print("The final price of the meal is $"+str(ans)+".")
